### ISSUE TITLE
Unable to pass data to input: shows up empty

### ISSUE THREAD

**Issue by n7229** [JsDaddy/ngx-mask#91]
lets say
sellingPrice = 2.22

data in the input field shows up empty

<input type="text" mask="0.00"  [(ngModel)]="sellingPrice"  />

**Comment by gotshoo** [comment]
Same here. I am using a reactive form and can see a value was patched to the form, but the value does not display in the input field. Validated the data matches the mask. 

![image](https://user-images.githubusercontent.com/1024817/36858524-004db384-1d41-11e8-8c2b-48cbd9e4f167.png)

![image](https://user-images.githubusercontent.com/1024817/36858615-3546268e-1d41-11e8-90a7-18d6948f89ac.png)

**Comment by sagarpdev6** [comment]
I have the same issue in angular reactive forms. The value exist but its not visible due to the mask. Also, the same thing works perfectly fine on another form.

UI:
![screen shot 2018-04-04 at 4 47 32 pm](https://user-images.githubusercontent.com/36165315/38333965-10edd776-3828-11e8-87fe-de1977a100c1.png)

Source:
<img width="870" alt="screen shot 2018-04-04 at 4 47 12 pm" src="https://user-images.githubusercontent.com/36165315/38334000-2b6f28e8-3828-11e8-9fdf-6391e1e845be.png">

@gotshoo Zip masking works perfectly fine for me.

**Comment by sagarpdev6** [comment]
@gotshoo Here's my zip code mask:
<input *ngIf="rForm.value.country == 'USA'" type="text" formControlName="zip" mask="99999" />

**Comment by gotshoo** [comment]
@sagarpdev6 Thanks. I'll try changing my mask from '00000' to '99999' and see if that works.

**Comment by natan88** [comment]
I have the same problem

**Comment by gotshoo** [comment]
I did some work on it this week to see if I could figure out the issue. Checking out the Develop branch, it looks like @NepipenkoIgor  possibly tried fixing this by adding an async task to pull the maskExpression. 

https://github.com/JsDaddy/ngx-mask/commit/f83223332b9a462ce9b4863bf185952a5a8f21c3

The problem is that the maskExpress still comes back as null. I hacked it a little bit and was able to prevent it from deleting the value. It would only show the mask if you modified the value of the input. I don't have any code to include because it's at work right now.

Would love to see this get fixed. I will try to dig in a little more next week.
